README.md

```diff
@@ -1 +1,25 @@
-# node-process-lab
\ No newline at end of file
+# Access Global and Process Variables Lab
+
+In this lab, you'll need to demonstrate how to access `global` and `process` variables.
+
+## Objectives
+
+1. Write a script that accesses global object
+1. Add logic to access process info
+1. Save and run the scripts
+
+## Introduction
+
```

Comment:
can we frame this lab with some sort of real world tie? its much more motivating for students to feel like theyre doing something they'll use in the real world. if we can frame this like they're a systems engineer or something that will make a student feel like theyre doing more than just practicing but solving a real world problem